src/envoy/http/jwt_auth/http_filter.cc

```diff
@@ -61,7 +62,7 @@ void JwtVerificationFilter::onDone(const JwtAuth::Status& status) {
   if (state_ == Responded) {
     return;
   }
-  if (status != JwtAuth::Status::OK) {
+  if (status != JwtAuth::Status::OK && !allow_missing_or_failed_) {
```

Comment:
JwtAuthentictor should be the one to use the config.   You can put this code in 

https://github.com/istio/proxy/blob/master/src/envoy/http/jwt_auth/jwt_authenticator.cc#L230

Reply:
Done.